src/mudlet.cpp

```diff
@@ -767,6 +768,11 @@ void mudlet::slot_close_profile_requested( int tab )
     mHostDockConsoleMap.remove(pH);
     mHostConsoleMap.remove(pH);
 
+    for( TToolBar* pTB : hostToolBarMap ) {
+        pTB->setAttribute( Qt::WA_DeleteOnClose );
+        pTB->close();
```

Comment:
I have wrapped the `TAction:mpToolBar` member which store the `TToolBar` instance for an action with the `QPointer<T>` and was thinking that would automatically be used where ever the pointer for that TToolbar is copied to, according to what Qt docs say about QPointer being usable by the standard pointer syntax.   
Does it not apply to lists or am I not understanding and making a bad assumption?   

I don't have a problem with making the QPointer change, just checking if my understanding is correct. :)